src/runner/test-run/index.js

```diff
@@ -34,6 +36,17 @@ export default class TestRun extends Session {
         this.nativeDialogsInfoTimeStamp = 0;
         this.stepsSharedData            = {};
 
+        this.enableScreenshots = !!this.opts.screenshotPath;
+        this.screenshotCreated = false;
+
+        if (this.enableScreenshots) {
+            if (!this.test.screenshotPath)
```

Comment:
Why we need to store `sreenshotPath` in test? And when it can be already specified?